src/ucs/arch/aarch64/cpu.h

```diff
@@ -94,6 +95,33 @@ static inline void ucs_arch_wait_mem(void *address)
                   : "Q"(address));
 }
 
+static inline void ucs_clear_cache(void *start, void *end)
+{
+#if HAVE___CLEAR_CACHE
```

Comment:
let's define "ucs_arch_clear_cache", that will always use the inline assembly, and a top-layer wrapper "ucs_clear_cache" which is common for all archs, and contains the "#if HAVE___CLEAR_CACHE" to avoid code duplication
also pls document this function is intended for self modifying code

Reply:
done, added arch_clear_cache implementation & docs